src/profiles.js

```diff
@@ -0,0 +1,77 @@
+import tv4 from 'tv4'
+import jsonlint from 'json-lint'
+import _ from 'lodash'
+
+import Registry from './registry'
+
+class Profiles {
+
+  constructor(remote = false) {
```

Comment:
Is it possible to make `Profiles` returning promise (all async stuff in constructor) but `Profiles` methods synchronous (`tv4.validateMultiple` seems sync)? We will be calling `profiles.validate` alot inside `DataPackage` class.